test/OrganisationRegistry.ElasticSearch.Tests/EnvVarIgnoreFactAttribute.cs

```diff
@@ -7,10 +7,10 @@ public class EnvVarIgnoreFactAttribute : FactAttribute
     {
         public EnvVarIgnoreFactAttribute()
         {
-            string envVar = "ES_TESTS";
+            const string envVar = "IGNORE_ES_TESTS";
             var env = Environment.GetEnvironmentVariable(envVar);
```

Comment:
Not your code, but the naming here is weird. `ignoreEsTests` maybe?

Reply:
om consistent te zijn met de andere env vars ?